autozone-poc/src/components/Signin/Signin.js

```diff
@@ -0,0 +1,55 @@
+import React, { Fragment}  from 'react';
+import {useForm} from 'react-hook-form';
+import axios from 'axios';
+import {Link} from 'react-router-dom';
+
+// import TextInput from '../../utlis/formComponents/InputWithLabel';
+// import InputWithLabel from "../../utlis/formComponents/InputWithLabel";
+import "./Signin.css";
+import Button from '../../utlis/button/Button';
+
+export default function Signin (){
+
+    const {register, handleSubmit, errors}=useForm();
+
+    const onSubmit=(data)=>{ 
+      axios.post("http://localhost:5000/user/signin",data).then(response=>{
+        console.log(response);
+      })
+    }
+  return(
+    <div>
+   
+      <div className="container h-100">
+        <div className="row h-100 align-items-center">
+          <div className="col-4"></div>
+          <div className="col-4 align-items-center signincontainer">
+            <div className="page-title text-center">
+              <h2 className="signinhead">Sign In</h2>
+              
+            <form onSubmit={handleSubmit(onSubmit)}>
+              <input className="form-control" type="email" name="emailid" id="emailid" placeholder="Enter email id" ref={register({required:true})}/>
```

Comment:
Please discuss with Gopal and resuse the input field component

Reply:
implemented and push in second commit